src/backends/tensorflow/savedmodel_backend.cc

```diff
@@ -133,8 +133,19 @@ SavedModelBackend::CreateTRTISTFModel(
     const DimsList& dims =
         (io.has_reshape()) ? io.reshape().shape() : io.dims();
 
-    RETURN_IF_ERROR(CompareDimsSupported(
-        Name(), io.name(), input->shape_, dims, Config().max_batch_size() > 0));
+    if (input->shape_->rank_ != 0) {
+      RETURN_IF_ERROR(CompareDimsSupported(
+          Name(), io.name(), input->shape_, dims, Config().max_batch_size() > 0));
+    }
+    else {
+      // Use input shape from model configuration if empty
```

Comment:
"The savedmodel doesn't specify a shape for the input so use the shape from the model configuration"

Reply:
Fixed comment.